### PR TITLE
CC -SP4- 101.8.2 - Polkadot API for add cab

### PR DESCRIPTION
**Pull request by Ayushi-knoldus**
What does this change do?
Add a polkadot API to add new cab

Any additional information for the reviewer to start
NA

How should this be manually tested?
We need window OS in which Rust is installed. then execute this rust program

Which Trello task and sub-task does this change relate to?
CC -SP4- 101.8.2 - Polkadot API for add cab

Are there any changes pending?
No

Does any team have to be notified of changes in this feature?
Yes

Definition of Done:

 - [ ] Is there >90% unit test code coverage?
 - [ ]  Does this PR add new dependencies? If so, please list out the same.
 - [ ] Will this feature require a new piece of infrastructure to be implemented?
 - [x] Is there appropriate logging included?
 - [x] Does the project compile ok?
 - [x] Have Clippy violations been fixed?
 - [x] Have Code is properly formatted?

### CODE REVIEW

File: app/src/index.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                    name: 'Hash',
+                    location: ('u32', 'u32')
+                },
+              }
+        });
+        
+        try{
+            let namePromise = digestMessage("Ayushi Dwivedi");
+            namePromise.then((car_no)=>{
+                api.tx.carpooling.addNewCab(15,{id: 15, car_no: car_no , location: (20,40), price: 30 , destination:(40,20)});
```
Comment (aman2909verma) [inline app/src/index.js:60]: submit the transaction

**Review by aman2909verma — CHANGES_REQUESTED** [review]

File: app/src/index.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        const alice = keyring.addFromUri('//Alice');
+        const types = JSON.parse(readFileSync('./types.json', 'utf8'));
+        const api = await ApiPromise.create({ provider: wsProvider,
+            types
+        });
+        
+        try{
+            let namePromise = digestMessage("Ayushi Dwivedi");
+            namePromise.then((car_no)=>{
+                const addCab = api.tx.carpooling.addNewCab(15,{id: 15, car_no: car_no , location: (20,40), price: 30 , destination:(40,20)});
```
Comment (aman2909verma) [inline app/src/index.js:50]: use square brackets for tuples.

**Review by aman2909verma — CHANGES_REQUESTED** [review]

**Review by ankit-mogha — APPROVED** [review]

**Review by aman2909verma — APPROVED** [review]

**Comment by ankit-mogha** [comment]
Please resolve merge conflicts.

**Review by aman2909verma — CHANGES_REQUESTED** [review]
Remove merge conflicts

**Pull request closed without merging**